source/dotnet/Library/AdaptiveCards.Templating/AdaptiveCardsTemplateSimpleObjectMemory.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using AdaptiveExpressions.Memory;
+using Newtonsoft.Json;
+
+namespace AdaptiveCards.Templating
+{
+    /// <summary>
+    /// Implements AEL's IMemory interface
+    /// </summary>
+    public class AdaptiveCardsTemplateSimpleObjectMemory : IMemory 
+    {
+        private SimpleObjectMemory simpleObjectMemory;
+        /// <summary>
+        /// Simple implementation of AEL.Memory.IMemory
+        /// It serves as an entry points to additional customazation and optimazation
```

Comment:
minor nit

```suggestion
        /// It serves as an entry points to additional customization and optimization
```

Reply:
Thanks, Paul. PR got auto merged. I will make this change.